rsts/conf.py

```diff
@@ -92,14 +97,32 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'sphinx_rtd_theme'
-html_static_path = ['_static']
 html_logo = "flyte_lockup_on_dark.png"
+html_static_path = ['_static']
```

Comment:
```suggestion
html_static_path = []
```

Reply:
if you don't do this you just get a warning every time you build.

Reply:
ok